src/components/VideoList/VideoList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import VideoCard from '../VideoCard';
+
+const VideoListContainer = styled.div`
+  text-align: center;
+`;
+
+export default function VideoList({ videos, handleVideoSelected }) {
+  if (!videos) {
+    return <div />;
+  }
+
+  const renderedVideos = videos.map((video) => {
+    return (
+      <VideoCard
+        key={Math.random()}
```

Comment:
And instead of using a `Math.random()` you could use something unique from the video. i.e. the etag from the video or the title itself. It is not a good practice because every re-render the `Math.random()` would change and the key performance won't work.